### PR TITLE
Move documentation site to package

### PR DESCRIPTION
**Pull request by Matt-Butler**
Moving documentation site to package.

@cerner/terra 

https://github.com/cerner/terra-ui/issues/73

### CODE REVIEW

File: packages/terra-badge/lib/Badge.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import 'node_modules/terra-mixins/src/terra-mixins';
+@import '~terra-mixins/src/terra-mixins';
```
Comment (Matt-Butler) [inline packages/terra-badge/lib/Badge.scss]: Update this in the generator

File: packages/terra-button/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "terra-button",
   "main": "lib/Button.js",
   "private": true,
-  "version": "0.2.0",
+  "version": "0.1.0",
```
Comment (Matt-Butler) [inline packages/terra-button/package.json]: https://github.com/cerner/terra-button/tree/v0.1.0

Button only has version 0.1.0 released.

File: packages/terra-site/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# Terra Site
```
Comment (Matt-Butler) [inline packages/terra-site/README.md]: I need to add more information here.

File: packages/terra-button/package.json (hunk 2 above)
Comment (bjankord) [inline packages/terra-button/package.json]: Good catch.

**Comment by zhongyn** [comment]
+1

**Pull request merged**